src/pages/starter-kit/checkout/development.md

```diff
@@ -101,3 +114,25 @@ Run end-to-end tests:
 ```bash
 aio app test --e2e
 ```
+
+## Troubleshooting
+
+#### AioCoreSDKError 403 Forbidden
+
+If you encounter the error `AioCoreSDKError [EventsSDKError]: [EventsSDK:ERROR_GET_ALL_PROVIDERS] Error: 403 - Forbidden` when creating an event provider, please follow these steps:
+
+1. Select the project and workspace you set up during the [initial configuration](./getting-started.md#initial-configuration).
+1. Ensure you have added the [I/O Management API](./getting-started.md#initial-configuration)
+1. Verify that you have the required permissions in the [Adobe Developer Console](https://developer.adobe.com/console) > **OAuth Server-to-Server** in the side-navigation menu > **Scope**:
+
+  ```bash
+  ["AdobeID","openid","read_organizations","additional_info.projectedProductContext","additional_info.roles","adobeio_api","read_client_secret","manage_client_secrets","event_receiver_api"]
+  ```
```

Comment:
What does this bash command have to do with the previous step? Is this the scope they should see? if so, that should be explicitly stated.

Reply:
I'll added `as follows:`